doc/modules/outlier_detection.rst

```diff
@@ -112,14 +107,33 @@ an ellipse. For more details on the different estimators refer to the example
 :ref:`sphx_glr_auto_examples_plot_anomaly_comparison.py` and the sections
 hereunder.
 
+.. figure:: ../auto_examples/images/sphx_glr_plot_anomaly_comparison_001.png
+   :target: ../auto_examples/plot_anomaly_comparison.html
+   :align: center
+   :scale: 50
+
+The following figure benchmarks outlier detection algorithms using ROC
+(Receiver Operating Characteristics) curve on fives different datasets. 
+:class:`ensemble.IsolationForest` outperforms
+:class:`neighbors.LocalOutlierFactor` because LOF is not meant to predict on
+a test set by default.
+
+.. figure:: sphx_glr_plot_anomaly_bench_001.png
```

Comment:
I think it should be something like this. The png will be generated by sphinx when building the doc
```suggestion
.. figure:: ../auto_examples/images/sphx_glr_plot_anomaly_bench_001.png
```

Reply:
actually I am not sure that we want to put the figure in the User Guide. I would remove it and just leave what you added in the Examples section.